azblob/sas_service.go

```diff
@@ -51,13 +60,35 @@ func (v BlobSASSignatureValues) NewSASQueryParameters(sharedKeyCredential *Share
 	}
 	startTime, expiryTime := FormatTimesForSASSigning(v.StartTime, v.ExpiryTime)
 
+	 /*
+	 http://www.thecodelesscode.com/case/84
```

Comment:
I haven't understood this comment. I don't understand which variable is being used for two things.

Reply:
This was a "messy" holdover from a previous change I'd made-- Initially I had NewSASQueryParameters merged for Identity SAS.

I'll remove this comment and include the swagger in a moment.